lib/content-services/src/lib/content-metadata/components/content-metadata/content-metadata.component.html

```diff
@@ -1,45 +1,91 @@
-<div class="adf-metadata-properties">
-    <mat-accordion displayMode="flat"
-                   [multi]="multi">
-        <mat-expansion-panel *ngIf="displayDefaultProperties"
-                             [expanded]="canExpandProperties()"
-                             [attr.data-automation-id]="'adf-metadata-group-properties'">
-            <mat-expansion-panel-header>
-                <mat-panel-title class="adf-metadata-properties-title">
```

Comment:
As this class is removed from html then we can remove styles for this class (adf-metadata-properties-title) from content-metadata.component.scss

Reply:
@DenysVuika yes but ContentMetadataHeaderComponent has own duplicated styles for this class so we can remove this from content-metadata.component.scss, please see below: 

<img width="1149" alt="image" src="https://github.com/Alfresco/alfresco-ng2-components/assets/115619721/c5e4223b-7c24-4d82-9dfb-8513ff1d0906">


<img width="1281" alt="image" src="https://github.com/Alfresco/alfresco-ng2-components/assets/115619721/cebf1fc6-4a3d-4f37-ba10-44d3abc573e6">


It's duplicated

Reply:
Removed